common/defs/solana/programs.json

```
@@ -1359,6 +1359,58 @@
       "name": "Token Program",
       "instruction_id_length": 1,
       "instructions": [
+        {
+          "id": 0,
+          "name": "Initialize Mint",
+          "is_multisig": false,
+          "parameters": [
+            {
+              "name": "decimals",
+              "type": "u8",
+              "optional": false
+            },
+            {
+              "name": "mint_authority",
+              "type": "pubkey",
+              "optional": false
+            },
+            {
+              "name": "freeze_authority",
+              "type": "pubkey",
```

```suggestion
              "type": "authority",
```
(same for Token2022)